### PR TITLE
Styling redo

### PR DESCRIPTION
**Pull request by aleciafb**

### CODE REVIEW

File: app/styles/footer.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  font-size: $txt-tiny;
+  img {
+    height: 20px;
+    padding-right: 20rem;
+  }
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+}
+* {
```
Comment (aleciafb) [inline app/styles/footer.scss]: remove

File: app/styles/colors.scss [hunk 2]
```diff
@@ -1,12 +1,14 @@
 $brand-color: brand_color_from_banner(); //Brand specific color
+$primary-color: rgba(0,0,0,0.6);
+$secondary-color: rgb(255,255,255);
 
-$brand-hue: hue($brand-color);
-$brand-saturation: saturation($brand-color);
-$brand-lightness: lightness($brand-color);
+// $brand-hue: hue($brand-color);
```
Comment (aleciafb) [inline app/styles/colors.scss]: add

File: app/components/banner-nav.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    let scrolled = false;
+
+    Ember.$(window).scroll(function () {
+      if (200 < $(window).scrollTop() && !scrolled) {
+        nav.addClass('fixed');
+        $('.banner__header').animate({top: 0}, 0);
+        scrolled = true;
+      }
+
+      if (200 > $(window).scrollTop() && scrolled) {
```
Comment (aleciafb) [inline app/components/banner-nav.js]: JSHint love

File: app/index.html [hunk 4]
```diff
@@ -12,7 +12,6 @@
     <link rel="stylesheet" href="assets/vendor.css">
     <link rel="stylesheet" href="assets/adlet.css">
 
-    <link rel="stylesheet" media="screen" href="https://fontlibrary.org/face/cabin" type="text/css"/>
     <link rel="stylesheet" media="screen" href="https://fontlibrary.org/face/fantasque-sans-mono" type="text/css"/>
```
Comment (aleciafb) [inline app/index.html]: remove

File: app/templates/application.hbs [hunk 5]
```diff
@@ -1,16 +1,14 @@
...
-</section>
+<div class="page-wrap">
+{{outlet}}
+</div>
+<footer class="footer">
+  <span class="footer__slogan">
+    powered by
+  </span>
+  <img class="footer__slogan__logo" src="assets/images/logo-dark.png">
+<!--   {{#if session.isAuthenticated}}
```
Comment (aleciafb) [inline app/templates/application.hbs]: ?????

File: app/components/banner-nav.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  didInsertElement() {
+    var nav = Ember.$('body');
```
Comment (A-Helberg) [inline app/components/banner-nav.js]: let 😏

**Pull request merged**